tests/integration/test_integration_vanila.py

```diff
@@ -31,7 +31,7 @@ async def test_integration_vanilla(self):
         c = Client(conjur_data, authn_strategy=authn_provider,
                    ssl_verification_mode=SslVerificationMode.INSECURE)
         resources = await c.list()
-        self.assertEqual(len(resources), 6)
+        self.assertGreater(len(resources), 0)
```

Comment:
Would this be better?
```suggestion
        self.assertGreater(len(resources), 6)
```